src/index.ml

```diff
@@ -128,11 +131,13 @@ module Make (K : Key) (V : Value) (IO : IO) = struct
   let clear t =
     let t = assert_open t in
```

Comment:
I know it's not strictly part of this PR, but could change this to `check_open` now? The `assert_*` prefix is now overloaded to mean two different things.

Reply:
Done :heavy_check_mark: